deploy/helm/templates/_helpers.tpl

```diff
@@ -376,3 +376,13 @@ YAML with the filter.
 {{- end -}}
 
 {{- end -}}
+
+{{/*
+Check whether the SWI endpoint check is enabled
+
+Usage:
+{{ isSwiEndpointCheckEnabled . }}
+*/}}
+{{- define "isSwiEndpointCheckEnabled" -}}
```

Comment:
I don't really like templates, but the condition was too complex and in too many places not to create a template for it.